packages/contracts-bedrock/test/Specs.t.sol

```diff
@@ -44,392 +56,736 @@ contract Specification_Test is CommonTest {
+            _auth: Role.NOAUTH,
+            _pausable: false
+        });
+        _addSpec({
+            _name: "L1StandardBridge",
+            _sel: _getSel("bridgeETH(uint32,bytes)"),
+            _auth: Role.NOAUTH,
             _pausable: false
         });
-        _addSpec(
-            "L1StandardBridge", _getSel("bridgeERC20To(address,address,address,uint256,uint32,bytes)"), false, false
-        );
-        _addSpec({ _name: "L1StandardBridge", _sel: _getSel("bridgeETH(uint32,bytes)"), _auth: false, _pausable: false });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("bridgeETHTo(address,uint32,bytes)"),
-            _auth: false,
+            _auth: Role.NOAUTH,
             _pausable: false
         });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("depositERC20(address,address,uint256,uint32,bytes)"),
-            _auth: false,
+            _auth: Role.NOAUTH,
             _pausable: false
         });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("depositERC20To(address,address,address,uint256,uint32,bytes)"),
-            _auth: false,
+            _auth: Role.NOAUTH,
+            _pausable: false
+        });
+        _addSpec({
+            _name: "L1StandardBridge",
+            _sel: _getSel("depositETH(uint32,bytes)"),
+            _auth: Role.NOAUTH,
             _pausable: false
         });
-        _addSpec({ _name: "L1StandardBridge", _sel: _getSel("depositETH(uint32,bytes)"), _auth: false, _pausable: false });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("depositETHTo(address,uint32,bytes)"),
-            _auth: false,
+            _auth: Role.NOAUTH,
             _pausable: false
         });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("deposits(address,address)"),
-            _auth: false,
+            _auth: Role.NOAUTH,
             _pausable: false
         });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("finalizeBridgeERC20(address,address,address,address,uint256,bytes)"),
-            _auth: true,
+            _auth: Role.BRIDGE,
             _pausable: true
         });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("finalizeBridgeETH(address,address,uint256,bytes)"),
-            _auth: true,
+            _auth: Role.BRIDGE,
             _pausable: true
         });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("finalizeERC20Withdrawal(address,address,address,address,uint256,bytes)"),
-            _auth: true,
+            _auth: Role.BRIDGE,
             _pausable: true
         });
         _addSpec({
             _name: "L1StandardBridge",
             _sel: _getSel("finalizeETHWithdrawal(address,address,uint256,bytes)"),
-            _auth: true,
+            _auth: Role.BRIDGE,
```

Comment:
Same for all the `Role.Bridge` calls here.

OTOH, I can see how you might mean that it's the 'other bridge' that's being authorized.

Reply:
But I think it's simplest to just use the heuristic of "what is the msg.sender check expecting".

Reply:
Gotcha. This makes sense. Thanks.

Reply:
Fixed.